src/R/Support/Impl/Help/Packages/PackageInfo.cs

```diff
@@ -80,7 +80,7 @@ internal sealed class PackageInfo : NamedItemInfo, IPackageInfo {
                 try {
                     var result = await _host.Session.PackagesFunctionsNamesAsync(Name, REvaluationKind.BaseEnv);
                     functions = result.Children<JValue>().Select(v => (string)v.Value);
-                } catch (TaskCanceledException) { } catch (REvaluationException) { }
+                } catch (RException) { }
```

Comment:
OK. but there is more that only `catch(RException)`

`AD7Property.SetValueAsStringWithError` - it is COM.
`RExecutionTracer`
`GridDataSource`
`REnvironmentProvider`

and several more